processing file asmFiles/beq1.asm (1 of 24)
processing file asmFiles/beq2.asm (2 of 24)
processing file asmFiles/bge1.asm (3 of 24)
processing file asmFiles/bge2.asm (4 of 24)
processing file asmFiles/bgeu1.asm (5 of 24)
processing file asmFiles/bgeu2.asm (6 of 24)
processing file asmFiles/blt1.asm (7 of 24)
processing file asmFiles/blt2.asm (8 of 24)
processing file asmFiles/bltu1.asm (9 of 24)
processing file asmFiles/bltu2.asm (10 of 24)
processing file asmFiles/bne1.asm (11 of 24)
processing file asmFiles/bne2.asm (12 of 24)
processing file asmFiles/count_days.asm (13 of 24)
processing file asmFiles/fib.asm (14 of 24)
processing file asmFiles/halt.asm (15 of 24)
processing file asmFiles/jal.asm (16 of 24)
processing file asmFiles/jalr.asm (17 of 24)
processing file asmFiles/mergesort.asm (18 of 24)
processing file asmFiles/mult.asm (19 of 24)
processing file asmFiles/mult_procedure.asm (20 of 24)
processing file asmFiles/beq1.asm (1 of 26)
--- memsim.hex	2025-09-20 14:56:00.731546800 -0400
+++ memcpu.hex	2025-09-20 14:56:00.701545643 -0400
@@ -1,16 +1,37 @@
-:0400000000200313C6
-:040001000020039345
-:04000200000008935F
-:0400030000010137C0
+:040000000020029347
+:04000100FFC1011327
+:040002000051202366
+:040003000060029304
 :04000400FFC1011324
-:04000500005008138C
-:040006000073086318
+:040005000051202363
+:040006000050029311
 :04000700FFC1011321
-:0400080001012023AF
-:04000900FFFFFFFFF7
-:04000A000010089347
-:04000B00FFC101131D
-:04000C00011120239B
-:04000D00FFFFFFFFF3
-:043FFE0000000001BE
+:040008000051202360
+:0400090000300313AD
+:04000A00001003934C
+:04000B000073086313
+:04000C00018000EF80
+:04000D00FFF30313E7
+:04000E00FF5FF06F31
+:04000F000001228347
+:040010000041011397
+:04001100FFFFFFFFEF
+:0400120000012E03B8
+:040013000041011394
+:0400140000012E8336
+:040015000041011392
+:0400160000000F13C4
+:04001700000E8E63E6
+:04001800001EFF9334
+:04001900000F8463ED
+:04001A0001CF0F33D0
+:04001B00001E1E1392
+:04001C00001EDE9351
+:04001D00FE9FF06FE3
+:04001E00FFC101130A
+:04001F0001E12023B8
+:0400200000008067F5
+:043FFD0000000005BB
+:043FFE000000001EA1
+:043FFF000000003C82
 :00000001FF
processing file asmFiles/beq2.asm (2 of 26)
processing file asmFiles/beq1.asm (1 of 24)
--- memsim.hex	2025-09-20 14:56:48.636392836 -0400
+++ memcpu.hex	2025-09-20 14:56:00.701545643 -0400
@@ -1,16 +1,37 @@
-:0400000000200313C6
-:040001000020039345
-:04000200000008935F
-:0400030000010137C0
+:040000000020029347
+:04000100FFC1011327
+:040002000051202366
+:040003000060029304
 :04000400FFC1011324
-:04000500005008138C
-:040006000073086318
+:040005000051202363
+:040006000050029311
 :04000700FFC1011321
-:0400080001012023AF
-:04000900FFFFFFFFF7
-:04000A000010089347
-:04000B00FFC101131D
-:04000C00011120239B
-:04000D00FFFFFFFFF3
-:043FFE0000000001BE
+:040008000051202360
+:0400090000300313AD
+:04000A00001003934C
+:04000B000073086313
+:04000C00018000EF80
+:04000D00FFF30313E7
+:04000E00FF5FF06F31
+:04000F000001228347
+:040010000041011397
+:04001100FFFFFFFFEF
+:0400120000012E03B8
+:040013000041011394
+:0400140000012E8336
+:040015000041011392
+:0400160000000F13C4
+:04001700000E8E63E6
+:04001800001EFF9334
+:04001900000F8463ED
+:04001A0001CF0F33D0
+:04001B00001E1E1392
+:04001C00001EDE9351
+:04001D00FE9FF06FE3
+:04001E00FFC101130A
+:04001F0001E12023B8
+:0400200000008067F5
+:043FFD0000000005BB
+:043FFE000000001EA1
+:043FFF000000003C82
 :00000001FF
processing file asmFiles/beq2.asm (2 of 24)
--- memsim.hex	2025-09-20 14:56:50.649470411 -0400
+++ memcpu.hex	2025-09-20 14:56:00.701545643 -0400
@@ -1,16 +1,37 @@
-:0400000000200313C6
-:040001000030039335
-:04000200000008935F
-:0400030000010137C0
+:040000000020029347
+:04000100FFC1011327
+:040002000051202366
+:040003000060029304
 :04000400FFC1011324
-:04000500005008138C
-:040006000073086318
+:040005000051202363
+:040006000050029311
 :04000700FFC1011321
-:0400080001012023AF
-:04000900FFFFFFFFF7
-:04000A000010089347
-:04000B00FFC101131D
-:04000C00011120239B
-:04000D00FFFFFFFFF3
-:043FFE0000000005BA
+:040008000051202360
+:0400090000300313AD
+:04000A00001003934C
+:04000B000073086313
+:04000C00018000EF80
+:04000D00FFF30313E7
+:04000E00FF5FF06F31
+:04000F000001228347
+:040010000041011397
+:04001100FFFFFFFFEF
+:0400120000012E03B8
+:040013000041011394
+:0400140000012E8336
+:040015000041011392
+:0400160000000F13C4
+:04001700000E8E63E6
+:04001800001EFF9334
+:04001900000F8463ED
+:04001A0001CF0F33D0
+:04001B00001E1E1392
+:04001C00001EDE9351
+:04001D00FE9FF06FE3
+:04001E00FFC101130A
+:04001F0001E12023B8
+:0400200000008067F5
+:043FFD0000000005BB
+:043FFE000000001EA1
+:043FFF000000003C82
 :00000001FF
processing file asmFiles/bge1.asm (3 of 24)
